Chem/Photolysis/Extend_HybridJ.m

```diff
@@ -0,0 +1,130 @@
+%ExtendHybridJ.m
```

Comment:
In the next release, will can add all of these to the standard J-value list. I will also include a tutorial and all code/solar spectra needed for users to generate their own "hybrid" J-values.